examples/calc.py

```diff
@@ -221,7 +223,7 @@ def keypress(self, size, key):
 
         if key == "backspace":
             raise ColumnDeleteEvent(self.letter, from_parent=True)
-        if key in list("0123456789"):
+        if key in string.digits:
```

Comment:
we can iterate over string directly without cast
we can use stdlib human-readable constant instead of copy-paste